src/ceaserCipher.js

```diff
@@ -0,0 +1,18 @@
+const alphabet = 'abcdefghijklmnopqrstuvwxyz';
+const cipher = (string, number) => {
+  const indexes = [];
+  string.split('').forEach((item) => {
+    if (alphabet.includes(item.toLowerCase())) {
+      if (alphabet.includes(item)) {
+        indexes.push(alphabet[(alphabet.indexOf(item) + number) % 26]);
```

Comment:
- [ ] [OPTIONAL] indexOf is not the fastest method, you could store the index of each alphabet letter in an object and have the lowercase letter be the key instead. That would make it faster:
ex: 
```
const alphabet = {
  'a': 0,
  'b': 1,
...
}

```